.pre-commit-config.yaml

```diff
@@ -711,6 +711,7 @@ repos:
         language: python
         pass_filenames: true
         files: ^scripts/ci/docker-compose/.+\.ya?ml$|docker-compose\.ya?ml$
+        exclude: ^scripts/ci/docker-compose/grafana/.|^scripts/ci/docker-compose/prometheus.yml
```

Comment:
I'm not thrilled with this solution but did not see a way to add an `ignore` file or tag.  If anyone knows a better way, I'm open to suggestions here.

Reply:
This is fine. We do not expect all files (especially kinda "external" to follow our liniting rules. We exclude a number of things.

Reply:
I spent a little while thinking about trying to use something equivalent to a .gitignore for this, but in the end I decided it's only two entries.  If we start adding more then we might want to consider a more scalable solution.  Or not.  Who knows.  But we're good with making that a problem for Future Us for now?